packages/notebook/src/widget.ts

```diff
@@ -2231,7 +2231,12 @@ export class Notebook extends StaticNotebook {
       // We don't want to prevent the default selection behavior
       // if there is currently text selected in an output.
       const hasSelection = (window.getSelection() ?? '').toString() !== '';
-      if (button === 0 && shiftKey && !hasSelection) {
+      if (
+        button === 0 &&
+        shiftKey &&
+        !hasSelection &&
+        !['input', 'option'].includes(target.localName)
```

Comment:
Would you mind using `tagName` instead of `localName`:

```suggestion
        !['INPUT', 'OPTION'].includes(target.tagName)
```

Reply:
Easy enough. Why is tagName preferred over localName?